components/21-atoms/file/_file.scss

```diff
@@ -71,9 +71,21 @@
   }
 
   +.help-text {
-    @include theme('color', 'color-zero--lighten-1', 'file-size-color');
+    @include theme('color', 'help-text-color', 'help-text-color');
     display: block;
     margin-top: .5rem;
     font-size: .7rem;
+    line-height: 1.2rem;
+  }
+}
+
+//TODO: Set to .help-text instead of .form-description in Drupal Theme
+.webform_document_file {
```

Comment:
Geen Drupal specifieke elementen gebruiken. De stijlgids is platform-onafhankelijk.
Alle elementen in de stijlgids worden generiek en algemeen opgebouwd (HTML + CSS). Alle CSS die hier voorkomt, is van toepassing op de twig file van deze component. In ons geval zal een Drupal theme verantwoordelijk zijn om dezelfde HTML markup te hanteren.